installers/binstall/scripts/nix/install.sh

```diff
@@ -15,7 +15,7 @@
 
 set -u
 
-BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/rover/releases/download"
+BINARY_DOWNLOAD_PREFIX="${APOLLO_ROVER_DOWNLOAD_GITHUB_HOST:="https://github.com/apollographql/rover/releases/download"}"
```

Comment:
It can be whatever. We had about 2 weeks of conversation to land on `APOLLO_ROVER_DOWNLOAD_GITHUB_HOST` because there are 2 different sources for apollo downloads; github and the apollo websites.  Some of the tools install from orbiter (apollo's website), and some download from github. 

There are already supported settings for the apollo based vars, and they use the `APOLLO_ROVER_DOWNLOAD_HOST`.

My original PR had this new user var set as `APOLLO_ROVER_BINARY_REMOTE`.

But it's not always a clear cut and dried when something is going to come from orbiter or github (rover for example downloading compose and supergraph plugins from apollo even if you installed rover using npm or the github bash script)

@Meschreiber and @shorgi had eventually settled on using `APOLLO_<ROVER|ROUTER>_DOWNLOAD_[GITHUB_?]HOST` as the dual pattern, so you could define a remote proxy for both source locations.

Reply:
That makes sense. I think that's all good then!

Also, apologies for missing the previous conversation.